aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -12,8 +13,9 @@ class ReviewComment:
     @staticmethod
     def from_json(data: Any) -> 'ReviewComment':
```

Comment:
**style**: Removing default values from .get() method calls could lead to KeyError exceptions. Consider keeping .get() with defaults or adding explicit error handling for missing keys.